packages/core/src/app.ts

```diff
@@ -1271,8 +1271,9 @@ export class AppBuilder<TreeNames> {
           }
 
           let runtime = join(packageName, name).replace(this.resolvableExtensionsPattern, '');
-          if (renamedModules && renamedModules[runtime]) {
-            runtime = renamedModules[runtime];
+          let runtimeRenameLookup = runtime.split('\\').join('/');
```

Comment:
runtime will look like: `ember-source\ember\index` because of the join.

renamedModules will contain items like:

```js
{
...
'ember-source/ember/index:'ember/index'
...
```
which fails to do the lookup. This converts the lookup to be in the expected format. I didn't change the meta info format as I'm not sure the full ramifications of that but a longer term solution would be to look into that.